torch/quantization/observer.py

```diff
@@ -273,7 +307,7 @@ def _calculate_qparams(
             zero_points: Zero points tensor of shape (#channels,)
         """
         if not check_min_max_valid(min_val, max_val):
-            return torch.tensor([1.0]), torch.tensor([0])
+            return torch.tensor([1.0]).to(min_val.device.type), torch.tensor([0]).to(min_val.device.type)
```

Comment:
would it work to specify the device at initialization, like `torch.tensor([1.0], device=min_val.device)`?

Reply:
yeah that would be better